src/game.js

```diff
@@ -0,0 +1,44 @@
+/* eslint-disable no-console */
+/* eslint-disable no-unused-vars */
```

Comment:
Remove the 'eslint-disable no-unused-vars' comment, as all variables in the code are actually used.